scalameta/semanticdb-scalac-core/src/main/scala/scala/meta/internal/semanticdb/SymbolOps.scala

```diff
@@ -1,84 +1,96 @@
 package scala.meta.internal
 package semanticdb
 
+import java.util.HashMap
 import scala.{meta => m}
 import scala.meta.internal.inputs._
 
 trait SymbolOps { self: DatabaseOps =>
 
+  private lazy val symbolCache = new HashMap[g.Symbol, m.Symbol]
```

Comment:
Can we maybe do this change without adding a 2 space indentation on the whole method? The diff makes it difficult to see the difference

Reply:
Would you like to have a separate commit that just changes indentation?